### PR TITLE
workflow: cucushift-installer-rehearse-azure-ipi-kubevirt-hypershift

### PR DESCRIPTION
**Pull request by LiangquanLi930**
workflow: cucushift-installer-rehearse-azure-ipi-kubevirt-hypershift

### CODE REVIEW

**Comment by LiangquanLi930** [comment]
@heliubj18 @jiajliu @kuiwang02 @liangxia PTAL this PR, thanks

**Comment by heliubj18** [comment]
LGTM, kubevirt hypershift install using dev's profile temporarily

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml [hunk 1]
```diff
@@ -249,6 +249,18 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-aws-usgov-ipi-private
+- as: e2e-azure-ipi-kubevirt-hypershift-mgmt-p2
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml:252]: Only add in 4.11 ? I would expect to add it in 4.12 (or both 4.11 and 4.12).

Comment (heliubj18) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml:252]: We will add another PR for 4.12. Currently, there is an [nlb bug](https://issues.redhat.com/browse/OCPBUGS-2946) blocking hypershift for OCP 4.12 install & creating on aws. We will investigate it on 4.12 azure & aws toghether for kubevirt hypershift.

**Comment by kuiwang02** [comment]
the updated file is not only under openshift-tests-private, so I can add lgtm, not say "iamok"
from my view, 
/lgtm

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml (hunk 1 above)
Comment (liangxia) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml:252]: Ack

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml [hunk 2]
```diff
@@ -249,6 +249,18 @@ tests:
...
+  cron: 0 4 */7 * *
+  steps:
+    cluster_profile: azure4
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml:255]: @jianlinliu  Is `azure4` the expected one here ?

Comment (heliubj18) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml:255]: We found only this profile could support kubevirt in prow. We will do more investigation here to apply qe's profile later.

Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml:255]: Is this urgent? Generally we are supposed to use QE's profile for testing.

Comment (heliubj18) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml:255]: Yes, we'd like to add it to have a look at the status of e2e.  It's also a part of HyperShift kubevirt plan. After this PR, we will continue to do more on kubevirt including replacing QE's profile & aws platform support.

Comment (heliubj18) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml:255]: Besides, it is easy to let kubevirt QE know the status of this pipeline and prepare their integration test based on this pipeline. Thanks!

Comment (LiangquanLi930) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml:255]: @jianlinliu could you help to review again? thanks^^

**Comment by kuiwang02** [comment]
the job fails to create hosted cluster.

**Comment by jianlinliu** [comment]
the job fails to create hosted cluster, will you continue debugging or a known issue to merge now?

**Pull request closed without merging**